frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -525,4 +531,17 @@ MyTreeNode _createMyTreeNode({required int arrLen}) {
   );
 }
 
+class DeepMatchesLooseInts extends CustomMatcher {
+  /// On WASM platforms, indexing into a [Int64List] or [Uint64List] will return
+  /// an Int64 from the fixnum library, which fails the type test. This asserts
+  /// that they only have to match by their values.
+  DeepMatchesLooseInts(matcher) : super("is a numeric", "value", matcher);
+  @override
+  Object? featureValueOf(actual) {
+    if (actual is Iterable) return actual.map(featureValueOf);
+    if (actual is int) return actual;
+    return int.tryParse('$actual');
```

Comment:
Not sure the type of "actual" here, but if it is a Int64, maybe we can use the Inte64.toInt() function